earth_enterprise/src/server/wsgi/serve/publish/publish_manager_helper.py

```diff
@@ -1417,9 +1417,9 @@ def _GetVhNameToBaseUrlDict(self):
       (scheme://host[:port]) -  {vh_name: vh_base_url}
     """
     vh_list = self.QueryVhList()
-    vhname_to_baseurl_dct = {
-        vh_name: self.GetVhBaseUrl(vh_url, vh_ssl) for (
-            vh_name, vh_url, vh_ssl) in vh_list}
+    vhname_to_baseurl_dct = []
```

Comment:
Q: I thought this was supposed to be a dictionary, rather than a list. Are the `vh_name`s consecutive integers?